src/app/services/auth.service.ts

```diff
@@ -4,9 +4,9 @@
  * Date: 12/18/15
  * Time: 10:34 AM
  */
-import {Injectable, EventEmitter} from "@angular/core";
-import {WindowService} from "./window.service";
-import {Http, Headers} from "@angular/http";
+import { Injectable, EventEmitter } from "@angular/core";
```

Comment:
I would prefer that the formatting stay consistent with the rest of the code; no spaces inside the braces.